File: scala3-springboot/src/main/scala/com/codegik/Application.scala
package com.codegik

import org.springframework.boot.SpringApplication.run
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.data.jdbc.repository.config.EnableJdbcRepositories

@SpringBootApplication
@EnableJdbcRepositories
class Application

object Application {

  def main(args: Array[String]): Unit = {
    run(classOf[Application], args *)
  }

}

File: springboot-webflux-sample/src/main/java/com/example/webflux/Application.java
package com.example.webflux;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan("com.*")
public class Application {
    public static void main(String[] args) {
//        System.setProperty("reactor.schedulers.defaultBoundedElasticOnVirtualThreads", "true");
        System.setProperty("reactor.schedulers.defaultBoundedElasticOnVirtualThreads", "true");
        System.setProperty("reactor.schedulers.defaultBoundedElasticSize", "200");
        System.setProperty("reactor.schedulers.defaultPoolSize", "200");
        SpringApplication.run(Application.class, args);
    }
}
